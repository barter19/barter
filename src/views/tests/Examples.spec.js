import { shallow } from '@vue/test-utils';
import ExamplesComponent from '../Examples';

const renderer = require('vue-server-renderer').createRenderer();

describe('Examples.vue Component', () => {
  it('renders to a nice snapshot', () => {
    // Mock Messages
    const messages = {
      en: {
        title: 'Title',
      },
    };

    const wrapper = shallow(ExamplesComponent, {
      i18n: {
        locale: 'en',
        messages,
      },
    });
    const { vm } = wrapper;

    renderer.renderToString(vm, (err, str) => {
      expect(str).toMatchSnapshot();
    });
  });
});
