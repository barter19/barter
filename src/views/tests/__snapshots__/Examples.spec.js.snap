// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples.vue Component renders to a nice snapshot 1`] = `
<main>
  <h2>Title</h2>
  <!---->
</main>
`;
